OptimizelySDKCore/OptimizelySDKCoreTests/OptimizelyTest.m

```diff
@@ -1182,6 +1193,123 @@ -(void)testGetEnabledFeaturesWithSomeFeaturesEnabledForUser {
     XCTAssertEqualObjects(features, enabledFeatures);
 }
 
+#pragma mark - TypedAudiences Tests
+
+- (void)testActivateWithTypedAudiences {
```

Comment:
I'd like to see tests for active using all of the typed audience (i.e. gt, lt, eq, substring, etc.)